src/features/usage-consolidation-accordion/usage-consolidation-content-resource.js

```diff
@@ -53,16 +60,19 @@ const UsageConsolidationContentResource = ({
         entityType={entityTypes.RESOURCE}
         year={year}
       />
-    )
-    : (
+    ) : (
       <>
         <SummaryTable
           id="resourceUsageConsolidationSummary"
+          entityType={entityTypes.RESOURCE}
           customProperties={customProperties}
+          noCostPerUseAvailable={noCostPerUseAvailable}
```

Comment:
You don't need this prop in `<Summary Table />`. `<NoCostPerUseAvailable />` is responsible for it